modules/infinite-scroll/infinity.js

```diff
@@ -411,6 +423,20 @@
 					// We don't want that, it breaks stuff, e.g. wp-mediaelement init.
 					script.async = false;
 
+					if ( item.after_handle ) {
+						script.onload = function() {
+							var after = document.createElement( 'script' ),
+								afterContent = document.createTextNode(
+									'//<![CDATA[ \n' + item.after_handle + '\n//]]>'
+								);
+
+							after.type = 'text/javascript';
+							after.appendChild( afterContent );
+
+							document.getElementsByTagName( elementToAppendTo )[ 0 ].appendChild( after );
+						};
+					}
```

Comment:
This code block appears 3 times in this file so it might be worth refactoring to a new function

Reply:
Good point, done in 9193a5dd6.